app/src/lib/databases/repositories-database.ts

```diff
@@ -48,6 +49,8 @@ export interface IDatabaseRepository {
   /** The last time the stash entries were checked for the repository */
   readonly lastStashCheckDate: number | null
 
+  readonly workflowPreferences: WorkflowPreferences
```

Comment:
I think this needs to be optional since we're not running any database upgrade/migration to ensure that it exists on every IDatabaseRepository

```suggestion
  readonly workflowPreferences?: WorkflowPreferences
```

Reply:
Thanks for the suggestion! this is something I messed up in Evelyn's PR 🤡 , gonna change it in a separate commit since it requires more changes outside of this file